### ISSUE TITLE
Suggestion: Spell Component: Force

### ISSUE THREAD

**Issue by HypoAkatsu** [Mithion/ArsMagica2#1558]
This spell would be very similar to telekinesis, except it doesn't move items, it moves mobs and players. The amount of mana to use this spell would depend on if the target is a player or a mob, how much HP they have, if they are a magical creature, or a non-magical creature, the spell will work better on a zombie than a mana elemental. Upon grabbing an enemy, you can toss them around, or maybe hit them into things like ceilings, walls, and floors for a certain amount of damage. Possibly the spell could be made so that you can only hold something for a certain amount of time, even if you had mana to possibly hold a victim for 5 minutes, you could only hold them for 30 seconds, until duration modifiers were added. I think this could be a good idea for a silver skill, rather than just being one unlockable from the start. So, maybe something like Spell Shape: Projectile/Channel; Components: Telekinesis, Gravity Well, Fling, Levitate, Slowfall; or at least maybe something like that to unlock it. I'm not really good at suggestions, but I hope I got the point across.

**Comment by malignantMystic** [comment]
Attract moves non-item entities.

**Comment by HypoAkatsu** [comment]
No, I mean as in freely move them, not just being able to either push or pull.

**Comment by malignantMystic** [comment]
Have you ever moved your mouse while using Telekinesis? Same principle. I've used Attract to keep things away from my house by throwing them sky-high, it's not as hard as you'd think.
